sides/roasted-sweet-potato-halves.md

```diff
@@ -0,0 +1,28 @@
+# Roasted sweet potato halves
+
+## Ingredients for two people
+
+- 2 large sweet potatoes (or 4-6 small sweet potatoes) cut in half
+- 1 medium white or yellow onion cut in quarters
+- 6 garlic cloves whole and peeled
+- Olive oil
+- Salt
+- Pepper
+- Spices like paprika and cumin
+
+
+## Supplies
+
+- Aluminum foil
+- Baking tray
+
+
+## Instructions
+
+1. Preheat the oven to 425°F.
+
+2. Coat all prepped veggies (sweet potato havles, onion quarters, and whole garlic cloves) with olive oil and salt, pepper, and spices to taste.
```

Comment:
```suggestion
2. Coat all prepped veggies (sweet potato halves, onion quarters, and whole garlic cloves) with olive oil and salt, pepper, and spices to taste.
```

Reply:
How much of each of these is recommended? "To taste" is nice, but giving readers a baseline to modify from is better.